snippets/language-ruby.cson

```diff
@@ -15,6 +15,9 @@
   'case … end':
     'prefix': 'case'
     'body': 'case ${1:object}\nwhen ${2:condition}\n\t$0\nend'
+  'begin … rescue … end':
+    'prefix': 'bre'
```

Comment:
That actually might be better. I used bre because originally I read the `if...else...end` snippet as `iee` like an abbreviation for the body.